### PR TITLE
test(audit): end-to-end audit tracker, mobile evidence, and settings/modal E2E coverage

### PR DESCRIPTION
**Pull request by chhoumann**
The deliverable for the full end-to-end MetaEdit audit. All code fixes already landed via #143, #144, #145; this branch carries the audit record and the broader E2E regression coverage that is not tied to a single fix.

## Contents
- **`AUDIT_TRACKER.md`** - the canonical spreadsheet: one row per user story (94 total) with Feature, User Story, Expected Behavior, Test/Errors/Fix/Retest status, and Desktop/Mobile verification. Generated from `audit/canonical-stories.json` + `audit/status.json` via `audit/generate-tracker.mjs`.
- **`AUDIT_NOTES.md`** - per-story code-derived expected behavior, edge cases, and pre-test risk hypotheses.
- **`AUDIT_ENVIRONMENT.md`** - desktop (isolated worktree Obsidian) and mobile (Android 15 / Obsidian 1.12.7 via CDP) harness setup + the mobile-sweep methodology.
- **`audit/`** - the inventory/bug-hunt artifacts, the tracker generator + status, and the mobile probe scripts (`mobile-sweep.js`, `mobile-ui.js`, `mobile-eval.sh`).
- **`tests/e2e/audit-settings.test.ts`, `tests/e2e/audit-modals.test.ts`** - live E2E coverage for the settings tab (toggles, gear-expand, persistence, Auto/Progress CRUD, Edit Meta menu registration, suggestion exclusion) and modal/kanban flows (autoprop pick + cancel, board mapping + lane display, native date picker).

## Result
- 94/94 stories Pass on desktop; 82 Pass + 12 N/A (desktop-only right-click menus) on mobile.
- Full suites green: 287 unit + 93 E2E (14 files).
- One flagged product limitation (AUTO-08, degenerate Multi-auto-property-for-tag-leaf, fails safely).

## Testing
`pnpm run lint && pnpm run build && pnpm run test` green; full `pnpm run test:e2e` green on a fresh isolated instance.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: audit/generate-tracker.mjs [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+			retest: "-",
+			desktop: "-",
+			mobile: "-",
+		};
+	}
+}
+
+const cell = (v) =>
+	String(v ?? "")
+		.replace(/\|/g, "\\|")
```
[the remark this answers is not in the document]
Comment (chhoumann) [reply audit/generate-tracker.mjs:44]: Fixed: `cell()` now escapes backslashes before escaping pipes, so the pipe-escape introducer is unambiguous.
